ui/src/styles/_layouts.scss

```diff
@@ -44,6 +47,9 @@
   &--center {
     --justifyContent: center;
   }
+  &--start {
```

Comment:
flex is start by default so this shouldn't be necessary

Reply:
do i need to add something else? when i take off flex--start the icon is centered

Reply:
I t is probably inheriting the class from something overriding it locally on the page where you set this. Try tracing and adjusting locally if possible, if that's not possible then we can look globally, maybe center is getting set accidentally somewhere globally